drivers/dma/dma_silabs_siwx91x.c

```diff
@@ -60,6 +63,18 @@ struct dma_siwx91x_data {
 					      */
 };
 
+static void siwx91x_dma_pm_policy_state_lock_get(void)
+{
+	pm_policy_state_lock_get(PM_STATE_SUSPEND_TO_IDLE, PM_ALL_SUBSTATES);
+	pm_policy_state_lock_get(PM_STATE_STANDBY, PM_ALL_SUBSTATES);
```

Comment:
The dtsi declare these power states:
```
               power-state-name = "runtime-idle";
               power-state-name = "suspend-to-idle";
```
So, `PM_STATE_STANDBY` -> `PM_STATE_RUNTIME_IDLE`.